File: Lab7_ListTravel/app/src/main/java/com/example/lab7_listtravel/MainActivity.java
package com.example.lab7_listtravel;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ListView;
import android.widget.TextView;

import java.util.ArrayList;
import java.util.List;

public class MainActivity extends AppCompatActivity {
    private List<Travel> listTravel;
    private AdapterTravel adapterTravel;
    private DatabaseHandler db;
    private ListView lvTravel;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        db = new DatabaseHandler(this);

        lvTravel = findViewById(R.id.lv_Travel);
        listTravel = new ArrayList<>();
        listTravel = db.getAllTravels();
        adapterTravel = new AdapterTravel(this,R.layout.item_travel, listTravel);
        lvTravel.setAdapter(adapterTravel);
        Button btnSave = findViewById(R.id.btnSave);
        TextView tvName = findViewById(R.id.tv_travel);
        btnSave.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                db.addTravel(new Travel(tvName.getText().toString()));
                loadListView();
            }
        });


    }
    private void loadListView(){
        List<Travel> list = new ArrayList<>();
        list = db.getAllTravels();
        adapterTravel = new AdapterTravel(this, R.layout.item_travel,list);
        lvTravel.setAdapter(adapterTravel);
    }
}

File: Lab1/app/src/main/java/com/example/lab1/MainActivity.kt
package com.example.lab1

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main);

        var btn_login = findViewById<Button>(R.id.btn_login);
        btn_login.setOnClickListener(View.OnClickListener {
            val intent = Intent(this, layout_login::class.java);
            startActivity(intent);
        })

        var btn_signup = findViewById<Button>(R.id.btn_signup);
        btn_signup.setOnClickListener(View.OnClickListener {
            val intent = Intent(this, layout_signup::class.java);
            startActivity(intent);
        })


    }

}